http/server.go

```diff
@@ -129,7 +129,7 @@ func GetSkipAndCount(r *http.Request) (skip int, count int, exists bool, err err
 		return skip, count, exists, err
 	}
 	// Cap skip at 10000
-	skip = int(math.Min(float64(skip), 10000))
+	skip = int(math.Min(float64(skip), math.MaxInt64))
```

Comment:
I'm not sure it's safe to increase this to 4 billion billion. What's the use-case where you'd skip that many rows? It seems a different design would be better in that case. Could we limit it to math.MaxInt32 or a billion?

Reply:
```
$ go doc math.MaxInt64  | grep Int64
	MaxInt64  = 1<<63 - 1
```

Reply:
Moved convo to slack